cdc/owner/changefeed.go

```diff
@@ -443,8 +443,8 @@ func (c *changefeed) tick(ctx cdcContext.Context, captures map[model.CaptureID]*
 		}
 	})
 
-	c.updateStatus(newCheckpointTs, newResolvedTs, minTableBarrierTs)
-	c.updateMetrics(currentTs, newCheckpointTs, metricsResolvedTs)
+	c.updateStatus(newCheckpointTs, newResolvedTs, barrier.minDDLBarrierTs)
```

Comment:
which one is the globalBarrierTs now?

Reply:
`updateStatus` only update status in etcd, however globalBarrierTs is updated by p2p, ref: https://github.com/pingcap/tiflow/pull/8964/files#diff-953746562776cf8aa41cd4a971df8c546649d2721ae787607aed2069f3b44eb0R353-R354